lbrynet/core/client/ClientProtocol.py

```diff
@@ -132,8 +132,8 @@ def send_request_or_close(do_request):
                 log.info(
                     "Closing the connection to %s due to having no further requests to send",
                     self.peer)
+                self.peer.report_success()
```

Comment:
this allows us to know that a peer connection and request/reply to it was  completed and successful